examples/bri-3/src/bri/zeroKnowledgeProof/capabilities/verifyCCSMAnchor/verifyCCSMAnchor.command.ts

```diff
@@ -0,0 +1,11 @@
+import { DocumentObject } from '../../models/document';
+import { ZeroKnowledgeProofVerificationInput } from '../../models/zeroKnowledgeProofVerificationInput';
+
+export class VerifyCCSMAnchorCommand {
+  constructor(
+    public readonly inputForProofVerification:
```

Comment:
We do not want to expect two completely different types as a single input param (inputForProofVerification). This is usually a sign that we need multiple commands. I would suggest we stick with simple hash existence verification in this command and open a separate github issue to first agree and then potentially implement additional verifications of the hash.